docs/program-setup/bare_metal/c_small_libs.md

```diff
@@ -1,5 +1,7 @@
 <h1 id="using-small-c-libraries">Using small C libraries in Mbed OS bare metal</h1>
-
+<!--is this page for app developers? do we expect them to always manually specify for their app that they want to use bare metal?-->
```

Comment:
At the moment the default for all targets is full Mbed OS. So users/applications need to explicitly enable the bare metal profile. This should change post Mbed 6.